src/config/dark-sites.config

```diff
@@ -1120,9 +1036,7 @@ statbot.net
 stateofjs.com
 status.codeberg.eu
 status.kyun.host
-status.pandadev.net
 steam.design
-steambase.io
```

Comment:
This one works for me and should stay here:
![obraz](https://github.com/user-attachments/assets/5d2b543f-dd1c-41bd-8755-725b1df85f42)